mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/mail/CassandraAnnotationMapper.java

```diff
@@ -64,6 +67,18 @@ public CassandraAnnotationMapper(CassandraId mailboxId, Session session) {
             .collect(Collectors.toList());
     }
 
+    public List<MailboxAnnotation> getAnnotationsByKeysWithOneDepth(Set<String> keys) {
```

Comment:
There is something wrong when one convert a set to a list. It means that somewhere types might be a Set, or a Collection, IMO.